### ISSUE TITLE
Principles of Management: unwanted comment line or processing instruction appears in between paragraph

### ISSUE THREAD

**Issue by brittany-johnson** [openstax/webview#2186]
@clteam18 commented on [Wed Jan 30 2019](https://github.com/openstax/cnx-recipes/issues/806)

# What happened?

In Chapter 14, under module **Process Theories of Motivation** unwanted comment line or processing instruction appears in between the paragraph as shown in screenshot.

# Environment:

- **Book(s)**: Principles of Management
- **URL/Server**: https://content02.cnx.org/contents/kAvMZRJp@3.1:uEJByd-U@2/1-3-Process-Theories-of-Motivation
- **Location (page number, chapter/section)**: Chapter 14 - m10639
- **Suggested nearby searchable text**: We may consider a 10 percent pay increase desirable until we find out that it was the lowest raise given in our work group.

# Screenshots

(_attach here (please clearly label what environment each screenshot came from)_)
![image](https://user-images.githubusercontent.com/39794953/51981857-4f08bb00-24ba-11e9-8ad7-cf0922d8adc4.png)

# Additional notes

 (_For example, steps to reproduce, related issues, more context, etc._)

---

@openstaxalina commented on [Wed Jan 30 2019](https://github.com/openstax/cnx-recipes/issues/806#issuecomment-458986151)

@marcjeannott this likely needs dev input, but could you please take a look at the markup and copy either the markup or a screenshot of the markup into this issue?  

---

@marcjeannott commented on [Mon Feb 04 2019](https://github.com/openstax/cnx-recipes/issues/806#issuecomment-459072714)

Ryan and I both couldn't see anything in the related paras
**Paragraph before the error**
`<term>Valences</term> are the easiest of the expectancy theory concepts to describe. Valences are simply the degree to which we perceive an outcome as desirable, neutral, or undesirable. Highly desirable outcomes (a 25 percent pay increase) are positively valent. Undesirable outcomes (being disciplined) are negatively valent. Outcomes that we’re indifferent to (where you must park your car) have neutral valences. Positively and negatively valent outcomes abound in the workplace—pay increases and freezes, praise and criticism, recognition and rejection, promotions and demotions. And as you would expect, people differ dramatically in how they value these outcomes. Our needs, values, goals, and life situations affect what valence we give an outcome. Equity is another consideration we use in assigning valences. We may consider a 10 percent pay increase desirable until we find out that it was the lowest raise given in our work group.`

**Paragraph after the error**
 `<emphasis effect="bold"><link target-id="ch14fig13" document=""/></emphasis> summarizes the three core concepts of expectancy theory. The theory states that our perceptions about our surroundings are essentially predictions about “what leads to what.” We perceive that certain effort levels result in certain performance levels. We perceive that certain performance levels result in certain outcomes. Outcomes can be <term>extrinsic</term>, in that others (our supervisor) determine whether we receive them, or <term>intrinsic</term>, in that we determine if they are received (our sense of achievement). Each outcome has an associated valence (outcome A’s valence is <m:math display="inline"><m:mrow><m:msub><m:mtext>V</m:mtext><m:mtext>a</m:mtext></m:msub></m:mrow></m:math>). Expectancy theory predicts that we will exert effort that results in the maximum amount of positive-valence outcomes.<footnote id="fs-idm406051536"><para id="fs-idm409782048">It can also be expressed as an equation:</para><equation id="fs-idm408251040" class="unnumbered"><label/><m:math display="block"><m:mrow><m:mtable columnalign="left"><m:mtr columnalign="left"><m:mtd columnalign="left"><m:mrow><m:mi>F</m:mi><m:mi>o</m:mi><m:mi>r</m:mi><m:mi>c</m:mi><m:mi>e</m:mi><m:mtext> </m:mtext><m:mtext>to</m:mtext></m:mrow></m:mtd><m:mtd columnalign="left"><m:mrow/></m:mtd><m:mtd columnalign="left"><m:mrow/></m:mtd></m:mtr><m:mtr columnalign="left"><m:mtd columnalign="left"><m:mrow><m:mtext>Choose</m:mtext></m:mrow></m:mtd><m:mtd columnalign="left"><m:mtext>=</m:mtext></m:mtd><m:mtd columnalign="left"><m:mrow><m:mtext>E1</m:mtext><m:mo>×</m:mo><m:mstyle displaystyle="true"><m:mo>∑</m:mo> <m:mrow><m:mrow><m:mo>(</m:mo><m:mrow><m:msub><m:mrow><m:mtext>E2</m:mtext></m:mrow><m:mtext>o</m:mtext></m:msub><m:mo>×</m:mo><m:msub><m:mtext>V</m:mtext><m:mtext>o</m:mtext></m:msub></m:mrow><m:mo>)</m:mo></m:mrow></m:mrow></m:mstyle></m:mrow></m:mtd></m:mtr><m:mtr columnalign="left"><m:mtd columnalign="left"><m:mrow><m:mtext>A level of Effort</m:mtext></m:mrow></m:mtd><m:mtd columnalign="left"><m:mrow/></m:mtd><m:mtd columnalign="left"><m:mrow/></m:mtd></m:mtr></m:mtable></m:mrow></m:math></equation><para id="fs-idm403642288">Where <m:math display="inline"><m:mrow><m:msub><m:mtext>V</m:mtext><m:mtext>o</m:mtext></m:msub></m:mrow></m:math> is the valence of a given outcome (o), and <m:math display="inline"><m:mrow><m:msub><m:mtext>E2</m:mtext><m:mtext>o</m:mtext></m:msub></m:mrow></m:math> is the perceived probability that a certain level of performance (e.g., Excellent, average, poor) will result in that outcome. So, for multiple outcomes, and different performance levels, the valence of the outcome and its associated performance➔outcome expectancy (E2) are multiplied and added to the analogous value for the other outcomes. Combined with the E1 (the amount of effort required to produce a level of performance), the effort level with the greatest <emphasis effect="italics">force</emphasis> associated with it will be chosen by the individual.</para></footnote> If our E1 or E2 is weak, or if the outcomes are not sufficiently desirable, our motivation to exert effort will be low. Stated differently, an individual will be motivated to try to achieve the level of performance that results in the most rewards.`

I also noticed that there's an emphasis tag on figure tags, this isn't necessary they're already bold. 

---

@openstaxalina commented on [Wed Jan 30 2019](https://github.com/openstax/cnx-recipes/issues/806#issuecomment-459131578)

Republished the module, collection rebaked, and confirming I still see the not converted yet message.

@helenemccarron @brittany-johnson @tomjw64 This is a new issue that came in after sprint planning yesterday.  This may be symptomatic of a larger issue.  We've seen these messages before; Ross is even working on quashing some now, so it would be good to coordinate with him:

*original issue:* https://github.com/openstax/rhaptos.cnxmlutils/issues/162
*PR fix for original issue:* https://github.com/openstax/rhaptos.cnxmlutils/pull/166
*similar issue; unsure how this was resolved, but it looks like the resolution was part of the crit sit fix:* https://github.com/openstax/webview/issues/2124
*similar issue that Ross is working on now:* https://github.com/openstax/webview/issues/2100 

---

@brittweinstein commented on [Thu Jan 31 2019](https://github.com/openstax/cnx-recipes/issues/806#issuecomment-459403248)

This is being resolved by another team. 

---

@openstaxalina commented on [Fri Feb 01 2019](https://github.com/openstax/cnx-recipes/issues/806#issuecomment-459779847)

Per conversation with @helenemccarron - I am reopening this for tracking.
@marcjeannott thank you for pasting in the markup.  Would you be able to paste in the markup _including the paragraph before the not converted yet message?_

---

@marcjeannott commented on [Mon Feb 04 2019](https://github.com/openstax/cnx-recipes/issues/806#issuecomment-460277306)

@openstaxalina good point! I've edited my comment above to show both paragraphs before and after the error. 

---

@openstaxalina commented on [Tue Feb 05 2019](https://github.com/openstax/cnx-recipes/issues/806#issuecomment-460718204)

Thanks - confirming that there is no content actually missing here.  The error message seems to appear even though no content is actually missing.

---

@openstaxalina commented on [Tue Feb 05 2019](https://github.com/openstax/cnx-recipes/issues/806#issuecomment-460726785)

The decision is to **hide the `NOT_CONVERTED_YET` message** for now, since content is not actually missing.
cc @helenemccarron @reedstrm

**Comment by brittany-johnson** [comment]
@openstaxalina Is it safe to assume that this should be applied to baked and unbaked books?

**Comment by openstaxalina** [comment]
@brittany-johnson I think so.  Sorry to kick this over, but @helenemccarron in your conversation with Ross, did you decide to apply this to baked _and_ unbaked books?

From my perspective, it would be okay to apply this to baked books only, if CNX is not comfortable making this change for unbaked books.

**Comment by helenemccarron** [comment]
@openstaxalina and @brittany-johnson, I don't think we decided either way. @reedstrm would you prefer to hide them only in baked books?

**Comment by openstaxalina** [comment]
cc @Bkblodget since this impacts non-OpenStax content (community created content).  

tl;dr: we decided to hide the processing instruction "NOT_CONVERTED_YET" because this will block us from publishing Principles of Management; however, there is no content actually missing, and we have not been able to track down why the processing instruction appears in this specific instance.  

The question is whether to hide the instruction for *all* content or only for baked (i.e. mainly OpenStax) books.

**Comment by openstaxalina** [comment]
@brittany-johnson we should hide the NOT_CONVERTED_YET message **for baked books only.**
cc @Stackblocks @helenemccarron @reedstrm @Bkblodget 

Source for decision: https://openstax.slack.com/archives/C0LA54Q5C/p1550078110083100

**Comment by brittany-johnson** [comment]
<img width="913" alt="screen shot 2019-02-18 at 10 20 12 am" src="https://user-images.githubusercontent.com/24909007/52963973-ce293a80-3366-11e9-996e-eb40af5d9275.png">

**Comment by openstaxalina** [comment]
Noted that we lost the space between paragraphs when the line is removed; @brittany-johnson posting new screenshot below.

**Comment by Stackblocks** [comment]
Displaying as specified with updated version of content / code. Figure number difference is due to difference in test chapter composition.

![image.png](https://images.zenhubusercontent.com/5af1f4c24b5806bc2bd257f2/0465149d-9585-4ef8-a6c6-dc2c5d87181f)
